HelloWorld/app/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "net.boondockradio.helloworld"
+        minSdkVersion 16
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
@eoinfogarty 
I agree.

@nbhd 
if you use many  `.jar`,write `compile fileTree(dir: 'libs', include: ['*.jar'])`.

if you use specific `hoge.jar`,write compile files('libs/hoge.jar')`.
see http://rakuishi.com/archives/5768/

This time you do not use the jar,unnecessary this line.